### ISSUE TITLE
Enhance positioning to allow us to specify a section name

### ISSUE THREAD

**Issue by sitefinitysteve** [Sitefinity/feather-widgets#978]
Title says it all, the 3 choices are nice, but would be better to let us enter a section, or AWESOMELY enumerate them and display in a dropdown to pick.

**Comment by PepiIvanova** [comment]
Hi Steve,

Could you please provide more details? Do you mean that the options in Javascript widget can be improved by adding a dropdown listing all sections in the layout file?

Thanks,
Pepi

**Comment by sitefinitysteve** [comment]
Yes please, or even just a property in an advanced view...just somewhere where we could specify an exact section name.
